### PR TITLE
Fix typos, update references to Mana, optimize PNGs

### PR DESCRIPTION
**Pull request by flodolo**
I was going through the repository to understand how to build and add content, and noticed a few issues.

My editor stripped a few whitespaces and added an empty line at the end of files (where missing), not sure if that's OK (e.g. I didn't spot any double space to force a line break).

One other thing I've noticed is the use of British (Canadian) spelling, especially in some documents, but I didn't touch that ;-)

P.S. I didn't go through the docs extensively, this is a superficial pass while I was trying to get a sense of the land.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -32,8 +32,8 @@ module.exports = {
           position: "right",
         },
         {
-          href: "https://mana.mozilla.org/wiki/display/FJT/Project+Nimbus",
```
Comment (flodolo) [inline docusaurus.config.js:35]: This URL doesn't redirect in Confluence, no clue if I've picked the right page.

File: docs/workflow/designing.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ Once you have concluded that an experiment is the correct approach for you conti
 
 ## To design an experiment
 
-1. Create an [experiment brief](https://docs.google.com/document/d/1eFGL9FATIuZudjSItpIT2Ct1C5qb5E3Qk7hJuJQT67s/edit#heading=https://docs.google.com/document/d/1eFGL9FATIuZudjSItpIT2Ct1C5qb5E3Qk7hJuJQT67s/edit)
+1. Create an [experiment brief](https://docs.google.com/document/d/1eFGL9FATIuZudjSItpIT2Ct1C5qb5E3Qk7hJuJQT67s/edit)
 
-1. File a [data org (DO) Jira ticket](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=109990007)
```
Comment (flodolo) [inline docs/workflow/designing.md:21]: This URL doesn't redirect (and it's broken even in a page of the Data Org).

**Comment by flodolo** [comment]
@jaredlockhart This might be for you to review when you're back.

**Comment by flodolo** [comment]
Also added a commit to optimize PNG images (folder went from 8.6 MB to 2.6 MB)

File: docs/jetstream/outcomes.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Experimenter will need to be re-deployed to pick up a new Outcome. Please ask in
 
 It is also possible to parameterize `select_expression` in outcomes, values the parameters to be replaced with can then be specified in external jetstream config.
 
-Example of an outcome using parametization:
+Example of an outcome using parametrization:
```
Comment (jaredlockhart) [inline docs/jetstream/outcomes.md:59]: `parameterization`

**Review by jaredlockhart — APPROVED** [review]
Wow @flodolo this is amazing!  Thank you!  Did you use a tool to do this or you did it all by hand?  Thank you!! ❤️

**Pull request merged**